ui/imports/utils/Utils.qml

```diff
@@ -371,8 +371,8 @@ QtObject {
             case "first":
                 if (firstPasswordField.text === "") {
                     return [false, qsTr("You need to enter a password")];
-                } else if (firstPasswordField.text.length < 6) {
-                    return [false, qsTr("Password needs to be 6 characters or more")];
+                } else if (firstPasswordField.text.length < 10) {
```

Comment:
Suggestion: You could use a parameter here to avoid typos.

Reply:
Good point. Will add a constant to be used in password views